src/api/java/CMakeLists.txt

```diff
@@ -9,8 +9,17 @@
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# copy cvc libraries to the resource directory
+file(GLOB CVC_LIBS "${CMAKE_INSTALL_PREFIX}/lib/*.so*")
```

Comment:
I actually see another issue here: this code is executed at configure time (i.e. when you run `configure.sh` or `cmake`). At this point in time, there may not even be any `*.so` files. And even if they exist (because we have built them at some point in the past) they may (and probably will) be outdated.
I think that all this code should go into a custom command or target (so that it is executed at build time), just like generating the Kind file or calling gradle.

Reply:
The jni includes cvc installed files and libraries. Since these are generated after `make install` which is separate from the default build, the java api is build is not run when `make` is executed for now.
The `.so` files are generated with the default build, however, the include files are not as far as I know.

Reply:
Yes, that's a good point. This should all be doable with generator expressions.

Reply:
Changed the cmake to use generater expressions.